Description: Use python3 during build
Author: Matthijs Kooijman <matthijs@stdin.nl>
Forwarded: not-needed
Bug-Debian: https://bugs.debian.org/967192
--- a/scripts/authorlist.py
+++ b/scripts/authorlist.py
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/python3
 #
 # This file is part of the OpenMSX music set for OpenTTD.
 # OpenMSX is free content; you can redistribute it and/or modify it under the terms of the
--- a/scripts/md5list.py
+++ b/scripts/md5list.py
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/python3
 #
 # This file is part of the OpenMSX music set for OpenTTD.
 # OpenMSX is free content; you can redistribute it and/or modify it under the terms of the
--- a/scripts/midifiles.py
+++ b/scripts/midifiles.py
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/python3
 #
 # This file is part of the OpenMSX music set for OpenTTD.
 # OpenMSX is free content; you can redistribute it and/or modify it under the terms of the
--- a/scripts/namelist.py
+++ b/scripts/namelist.py
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/python3
 #
 # This file is part of the OpenMSX music set for OpenTTD.
 # OpenMSX is free content; you can redistribute it and/or modify it under the terms of the
--- a/scripts/playlist.py
+++ b/scripts/playlist.py
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/python3
 #
 # This file is part of the OpenMSX music set for OpenTTD.
 # OpenMSX is free content; you can redistribute it and/or modify it under the terms of the
--- a/scripts/sanitize_list.py
+++ b/scripts/sanitize_list.py
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/python3
 #
 # This file is part of the OpenMSX music set for OpenTTD.
 # OpenMSX is free content; you can redistribute it and/or modify it under the terms of the
